pokemongo_bot/__init__.py

```diff
@@ -160,10 +112,55 @@ def _setup_api(self):
         print('[#] Stardust: {}'.format(stardust))
         print('[#] Pokecoins: {}'.format(pokecoins))
         self.get_player_info()
-        print('[#]')
 
+        if self.config.firsttrans:
+            self.first_transfer()
+            
+        print('[#]')
         self.update_inventory();
-
+        
+    def first_transfer_get_groups(self):
+        self.pokemon_groups = {}
```

Comment:
@necrozis you mind accumulating in `pokemon_groups`(without `self`) and then returning it in the end? Also please add an `_` to the beginning of the name of this method (it indicates it's a private method) and move it below the `first_transfer` method.